### DISCUSSION TITLE
Problem using DisplayAlert

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#24370] [Q&A]
I need to show a display allert if a value is zero; I have to call it from a sync method, so I do:
  

```
Task.Run(async () =>
{
    await ShowAlertAsync();
}).Wait();
```

```
public async Task ShowAlertAsync()
  {
      try
      {
        
          await DisplayAlert("Error", "Value can't be.", "OK");
      }
      catch (Exception ex)
      {
          await DisplayAlert("Error", $"An error occurred: {ex.InnerException?.Message ?? ex.Message}", "OK");
      }
  }
```

but I get:  "One or more errors occurred. (Exception has been thrown by the target of an invocation.)" without any details.
Anyone can help please?

**Comment by gautambjain** [comment]
Why do you want to use Task.Run()? Did you try calling ShowAlertAsync() directly?
